models/user.server.model.js

```diff
@@ -1,13 +1,15 @@
-const User = require('../config/db_schemas/user.schema');
+const
+    User = require('../config/db_schemas/user.schema'),
+    saltedMd5 = require('salted-md5');
 
 /**
  * Hashes a given plaintext password
  * @param password plaintext password for hashing
  * @returns {String} hashed password
  */
-function hashPassword(password) {
-    // TODO: Create a function for hashing (and maybe salting) passwords
-    return password;
+hashPassword = function(password) {
+    const hashedPassword = saltedMd5(password, 'UniForum-Salt')
+    return hashedPassword;
```

Comment:
```suggestion
    return saltedMd5(plaintextPassword, 'UniForum-Salt');
```
This can be simplified

Reply:
Done.